stdlib/public/core/ArrayShared.swift

```diff
@@ -140,11 +140,14 @@ internal func _growArrayCapacity(_ capacity: Int) -> Int {
 
 @_alwaysEmitIntoClient
 internal func _growArrayCapacity(
-  oldCapacity: Int, minimumCapacity: Int, growForAppend: Bool
+  oldCapacity: Int, minimumCapacity: Int, elementSize: Int, growForAppend: Bool
 ) -> Int {
   if growForAppend {
     if oldCapacity < minimumCapacity {
-      // When appending to an array, grow exponentially.
+      if minimumCapacity * elementSize <= 128 {
```

Comment:
What's magic about 128?

Reply:
Chosen by eyeballing a histogram of string lengths (I was originally doing this change for string, I should probably get a histogram of array counts to confirm it's still reasonable).